ginivision/src/main/java/net/gini/android/vision/internal/camera/photo/Photo.java

```diff
@@ -301,6 +329,8 @@ public int hashCode() {
         result = 31 * result + mRotationForDisplay;
         result = 31 * result + (mContentId != null ? mContentId.hashCode() : 0);
         result = 31 * result + mRotationDelta;
+        result = 31 * result + (mDeviceOrientation != null ? mDeviceOrientation.hashCode() : 0);
+        result = 31 * result + (mDeviceType != null ? mDeviceType.hashCode() : 0);
         return result;
```

Comment:
Why is it multiplied by 31?

Reply:
Could be any number, I let Android Studio generate these.